src/pages/chat.tsx

```diff
@@ -0,0 +1,179 @@
+import type { NextPage } from 'next';
+import Image from 'next/image';
+import avatarMale from '../images/avatar/male.png';
+import Chat from '../components/chat';
+import ChatBox from '../components/chatbox';
+
+const ChatPage: NextPage = (props): JSX.Element => {
+    return (
+        <div>
+            <div className="h-screen w-screen bg-gradient-to-r from-complementary to-dominant flex justify-center items-center">
+                <div className="p:2 w-screen h-screen md:h-3/4 md:w-4/5 md:rounded-2xl lg:bg-white lg:rounded-2xl chat-lg justify-between flex flex-col bg-white">
+                    <div className="flex sm:items-center justify-between py-3 border-b-2 border-gray-200 pr-3">
+                        <div className="relative flex items-center space-x-4">
+                            <div className="relative pl-6 flex items-center">
+                                <div className="w-full h-full sm:w-16 sm:h-16 flex items-center">
+                                    <Image
+                                        src={avatarMale}
+                                        alt="Male Avatar"
+                                        className="avatar"
+                                    />
+                                </div>
+                            </div>
+                            <div className="flex flex-col leading-tight">
+                                <div className="text-2xl mt-1 flex items-center">
+                                    <span className="text-gray-700 mr-3">
+                                        StinkyGiraffe#7468
```

Comment:
Almost missed this one but can we load the session in the page and if there is a session, can we put the user's name here?